### ISSUE TITLE
Is it possible to disable display of branches in the web build?

### ISSUE THREAD

**Issue by danielbachhuber** [composer/satis#393]
We're using Satis right now to build the [WP-CLI Package Index](http://wp-cli.org/package-index/), and would like to disable the display of branches in the built web HTML:

![image](https://cloud.githubusercontent.com/assets/36432/22701776/802cd2ba-ed13-11e6-9ed9-7334dd3ef547.png)

I saw the `skip-dev` argument for generating archives. Is there an equivalent way to only display tagged releases?

Thanks in advance!

From https://github.com/wp-cli/wp-cli/issues/3701

**Comment by alcohol** [comment]
Yeah you could limit the requirements to something other than '*'...

**Comment by aaukt** [comment]
In your satis configuration you can set the "minimum-stability" to "stable" to only see tagged releases.
The option defaults to "dev" in satis (where in composer it defaults to stable) that's why you all the branches.

https://github.com/composer/satis/blob/master/src/Console/Command/BuildCommand.php#L68

**Comment by alcohol** [comment]
That will work too, but then you will no longer get `dev-master` either.

**Comment by danielbachhuber** [comment]
I've worked out that `dev-master || >=0.0.0` is the requirement I need for packages.

Is it possible to set this requirement globally? We're [using a script](https://github.com/wp-cli/package-index/blob/master/scripts/prepare-satis.php) to generate `satis.json` from a list of URLs. While the package names most commonly correlate with their URLs, it's not guaranteed, so I'm not sure I can easily set the requirement for each package.
